1.0/docs/devguide/templates.md

```diff
@@ -277,6 +278,11 @@ different name for the index property.
     </template>
     {% endraw %}
 
+### Forcing synchronous renders {#synchronous-renders}
+
+Call [`render`](/{{versionPrefix}}/api/#dom-repeat:method-render) to force
+a `dom-repeat` template to synchronously render any changes to its data. 
+Normally changes are batched and rendered asynchronously.
```

Comment:
We should add the two use cases here:

https://github.com/Polymer/docs/issues/1439#issuecomment-158156925

Hmm... the first one isn't really a use case, it's more of a general description. I think you could replace it with these two:

- For unit testing, to ensure items have rendered before checking the generated DOM.
- To ensure a list of items have rendered before scrolling to a specific item.

And explicitly note that it does _not_ cause the dom-repeat to pick up changes to the array that were made without the Polymer array mutation methods. If you (or a third-party library) has mutated the array without using the Polymer array mutation methods, you need to use notifySplices (link to section). 

This is one of the #1 things that confuses people about using render.